sktime/base.py

```diff
@@ -16,7 +17,7 @@ def __repr__(self):
             name=name, params=pprint_sklearn(self.get_params(), offset=len(name), )
         )
 
-    def get_params(self):
+    def get_params(self, deep=False):
```

Comment:
sorry if i'm being an idiot here but where is the part where `deep` is incorporated into what the function does?

Reply:
nothing wrong with that question, it is just to be API compliant with scikit-learn. Otherwise (if used in conjunction with scikit-learn methods) ducktyping will fail.

Reply:
ok, so it's correct that there's nothing here that uses any kind of `deep` parameter?

Reply:
yes, it is just to get the signature of the method right

Reply:
got it i think it was just a little confusing that `deep=True` somewhere down below

Reply:
i agree, this is the sklearn implementation though. I guess if you want to set parameters (which is where `deep=True` comes up) then you want to set them all the way

Reply:
got it. you could add a comment to clarify but obv not essential